react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -122,17 +122,27 @@
 
 .firstItemInGroup {
   @media @mobile {
-    padding-top: @grid-row-height * 2;
+    padding-top: @grid-row-height * 3;
     position: relative;
     &::before {
       content: '';
       position: absolute;
       top: @grid-row-height;
-      right: 0;
-      left: @grid-gutter-width;
+      width: 100%;
       height: 1px;
       background: @sk-white;
       opacity: 0.2;
     }
   }
 }
+
+.mobileSigninSignout {
+  margin-top: @grid-row-height;
+  height: @grid-row-height * 9;
+
+}
+
+.authLoader {
+  .loader(4px, 3px, -4px (@grid-gutter-width * 1.5) 0 0);
```

Comment:
Rather than duplicating the markup and CSS, what do you think of adding an `inline` prop to the `Loader` component? Then we could also use it for the `loading` state in the `Button` component. Thoughts?

Reply:
Adding this now 👍